### PR TITLE
anaconda-cli-base v0.4.0

### PR DESCRIPTION
**Pull request by mattkram**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by mattkram** [comment]
@conda-forge-admin, please rerender

**Comment by AlbertDeFusco** [comment]
I think we test against 3.12

**Comment by mattkram** [comment]
> I think we test against 3.12

Good catch. Do we want to be setting an upper bound? I wouldn't really expect any breaking changes from python itself but also not sure of best practice.

**Comment by AlbertDeFusco** [comment]
I've stopped setting upper bounds

File: recipe/meta.yaml [hunk 1]
```diff
@@ -23,9 +23,11 @@ requirements:
     - pip
   run:
     - python >=3.8,<3.12
```
Comment (mattkram) [inline recipe/meta.yaml]:
```suggestion
    - python >=3.8
```

**Pull request merged**